src/sentry/models/externalintegrationproject.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    BoundedPositiveIntegerField, Model
+)
+
+
+class ExternalProjectIntegration(Model):
+    __core__ = False
+
+    organization_integration_id = BoundedPositiveIntegerField(db_index=True)
+    date_added = models.DateTimeField(default=timezone.now)
+    name = models.CharField(max_length=128)
+    external_id = models.CharField(max_length=64)
+    resolved_status = models.CharField(max_length=64)
+    unresolved_status = models.CharField(max_length=64)
```

Comment:
Wasn't sure about the lengths on these. Any suggestions?

Reply:
In Postgres, a text column, and a varchar are technically the same thing. You can think of a varchar simply as a text + a constraint. There's no efficiency gain by using a varchar aside from enforcing a length. So if you don't particularly care, you can just feel free to make them `TextField()` and not worry about the length.

If it's user import, can alwyas just restrict the length in the form validation or input data validation at that point, and keep it looser in the database. It'll avoid needing to change it later if say, 64 characters is not enough, etc.

Just generic knowledge sharing, I don't have a lot of context on what these are. :)